lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+    "\n",
+    "> $V$ is what [`scipy.optimize.minimize`](https://docs.scipy.org/doc/scipy-0.18.1/reference/generated/scipy.optimize.minimize.html) returns in its `hess_inv` field if you pass it the negative log likelihood."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Wording\n",
+    "\n",
+    "* Frequentist confidence intervals are different from Bayesian credible regions:\n",
+    "\n",
+    "> \"68% of datasets would give a 68% frequentist confidence interval that contains the true parameter value\"\n",
+    "\n",
+    "> \"The probability of the true parameter value lying within the 68% Bayesian credible region is 68%\"\n",
+    "\n",
+    "* The difference in wording comes from the two different definitions of probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Uncertainties in the estimators\n",
+    "\n",
+    "* The covariance matrix of a Gaussian approximation to the likelihood defines a 1-sigma, 2D, elliptical, _frequentist confidence region_\n",
+    "\n",
+    "* Since this came from transforming the sampling distribution, which is a PDF over datasets, the confidence interval enables conclusions in terms of fractions of an ensemble of (hypothetical) datasets\n",
+    "\n",
+    "* The 68% confidence interval is the region that we expect to contain the true parameter value 68% of the time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism and Bayesianism\n",
+    "\n",
+    "* In frequentism, the data are considered to be random variables (in large sets of hypothetical trials described with probability distributions) while parameters are considered fixed (and to be estimated)\n",
+    "\n",
+    "* In Bayesianism, the data are considered to be fixed (as constants, in datafiles) while parameters are considered random variables (to be inferred, with uncertainty described by probability distributions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism and Bayesianism\n",
+    "\n",
+    "Given an assumed model:\n",
+    "\n",
+    "* Frequentists seek to _transform_ the frequency distribution of the data into a frequency distribution of their estimators, and hence quantify their uncertainty in terms of _what they expect would happen if the observation were to be repeated_  \n",
+    "  \n",
+    "* Bayesians seek to _update their beliefs_ about their model parameters, and hence quantify their uncertainty in terms of _what might have been had the observation been different_, and _what they knew before the data were taken_\n",
```

Comment:
beliefs about -> knowledge of

Reply:
OK fine - makes sense if they have not been introduced to probability as quantifying degree of belief.